[2025-08-06T02:34:16.490+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-06T02:34:16.501+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T02:34:14.336602+00:00 [queued]>
[2025-08-06T02:34:16.507+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T02:34:14.336602+00:00 [queued]>
[2025-08-06T02:34:16.507+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-06T02:34:16.517+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_holiday_events> on 2025-08-06 02:34:14.336602+00:00
[2025-08-06T02:34:16.534+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=783) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-06T02:34:16.540+0000] {standard_task_runner.py:63} INFO - Started process 789 to run task
[2025-08-06T02:34:16.536+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_s3_metadata_dag', 'extract_holiday_events', 'manual__2025-08-06T02:34:14.336602+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/extract/mysql_to_s3_dag.py', '--cfg-path', '/tmp/tmp83zs5hch']
[2025-08-06T02:34:16.542+0000] {standard_task_runner.py:91} INFO - Job 34: Subtask extract_holiday_events
[2025-08-06T02:34:16.561+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-06T02:34:16.595+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T02:34:14.336602+00:00 [running]> on host 3bffe12d583b
[2025-08-06T02:34:16.661+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_s3_metadata_dag' AIRFLOW_CTX_TASK_ID='extract_holiday_events' AIRFLOW_CTX_EXECUTION_DATE='2025-08-06T02:34:14.336602+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-06T02:34:14.336602+00:00'
[2025-08-06T02:34:16.664+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-06T02:34:16.680+0000] {base.py:84} INFO - Using connection ID 'mysql_local' for task execution.
[2025-08-06T02:34:16.696+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:243 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-08-06T02:34:16.729+0000] {logging_mixin.py:188} INFO - Extracted holiday_events to /opt/***/temp/holiday_events.csv
[2025-08-06T02:34:16.730+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/temp/holiday_events.csv
[2025-08-06T02:34:16.733+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-06T02:34:16.754+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_s3_metadata_dag, task_id=extract_holiday_events, run_id=manual__2025-08-06T02:34:14.336602+00:00, execution_date=20250806T023414, start_date=20250806T023416, end_date=20250806T023416
[2025-08-06T02:34:16.762+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-06T02:34:16.778+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-06T02:34:16.779+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
